src/Cache/Engine/RedisEngine.php

```diff
@@ -116,6 +117,10 @@ protected function _connect(): bool
                 );
             }
         } catch (RedisException $e) {
+            if (class_exists(Log::class)) {
+                Log::error('RedisEngine RedisException: ' . $e->getMessage());
```

Comment:
```suggestion
                Log::error('RedisEngine could not connect. Got error: ' . $e->getMessage());
```

Reply:
Agreed, this is strictly related to establishing a connection.